cmd/object-handlers.go

```diff
@@ -661,6 +664,12 @@ func (api objectAPIHandlers) PutObjectPartHandler(w http.ResponseWriter, r *http
 		return
 	}
 
+	// X-Amz-Copy-Source shouldn't be set for this call.
+	if _, ok := r.Header["X-Amz-Copy-Source"]; ok {
```

Comment:
There is a reason to do that is "Get()" is not useful it would still return empty string for both cases when

"X-Amz-Copy-Source" is specified with empty string or "X-Amz-Copy-Source" is not specified. 

In this scenario we need to know if there is a header only not the value, so this is a correct style.